ruby/introduction/installing_ruby.md

```diff
@@ -125,6 +125,12 @@ ruby 3.1.2pxx (20xx-xx-xx revision xxxxx) [x86_64-linux]
 ~~~
 where x represents the version available at the time you installed Ruby.
 
+If you've already installed `rbenv`, we'll need to update the `ruby-build` plugin. As such, let's pull in the latest Ruby version definition:
```

Comment:
I would consider softening the language a bit from `need` to `may need` or similar, because it will depend on when they installed rbenv and using which method. Not that it really hurts to do it anyway.

Looking now, I think it may be better to put this either under a `troubleshooting` or `updating a previous install` (not married to these names, just examples) and make it explicit what problem it's trying to fix. Otherwise it's an extra step without context, and could possibly be missed. Perhaps even add an example of the potential error. What do you think?

Reply:
I agree with the suggested changes. Here's the corresponding commit:

https://github.com/TheOdinProject/curriculum/pull/24293/commits/df2503be0b60e867838f094e7c85a8bd2be473d9